utils.py

```diff
@@ -23,7 +23,7 @@
         import simplejson as json
         from simplejson import load as jload
         from simplejson import dump as jdump
-    except:
+    except BaseException:
```

Comment:
A better name for the import failure exception can be obtained by failing an import interactively.